src/test/java/net/revelc/code/impsort/LineEndingEdgeCasesTest.java

```diff
@@ -97,10 +102,10 @@ public void testPartiallyParsedFile() throws IOException {
         + "        System.out.println(\"Hello, world!\")\n" + "    }\n" + "}\n";
     Path p = new File(folder, "InvalidFile.java").toPath();
     Files.write(p, s.getBytes(UTF_8));
-    ImpSortException e = assertThrows(ImpSortException.class,
-        () -> new ImpSort(UTF_8, eclipseDefaults, true, true, LineEnding.AUTO).parseFile(p));
+    ImpSortException e = assertThrows(ImpSortException.class, () -> new ImpSort(UTF_8,
+        eclipseDefaults, true, true, LineEnding.AUTO, new SystemStreamLog()).parseFile(p));
     assertTrue(e.getReason() == Reason.PARTIAL_PARSE);
-    assertEquals("file: " + p + "; reason: the Java file contained parse errors", e.getMessage());
+    assertTrue(e.getMessage().contains("file: " + p + "; reason: the Java file contained parse errors"+System.lineSeparator()+"errorMessage: (line 3,col 43) Parse error. Found \"}\", expected one of  \"%=\" \"&=\" \"*=\" \"++\" \"+=\" \"--\" \"-=\" \"/=\" \";\" \"<<=\" \"=\" \">>=\" \">>>=\" \"^=\" \"|=\""));
```

Comment:
Checking for this specific error is going to be flaky as we update the parser and they change their error messages to fix typos, add new language features, etc. We could probably be okay with a less detailed check here, and in the other test cases below.

Reply:
Yes, I agree with you. 
We could just check some key word about error.